lbrynet/wallet/transaction.py

```diff
@@ -72,6 +72,15 @@ class Transaction(BaseTransaction):
     input_class = Input
     output_class = Output
 
+    @classmethod
+    def pay(cls, amount: int, address: bytes, funding_accounts: List[Account], change_account: Account):
+        ledger = cls.ensure_all_have_same_ledger(funding_accounts, change_account)
+        hash_of_address = ledger.address_to_hash160(address)
+
+        output = Output.pay_pubkey_hash(amount, hash_of_address)
+
+        return cls.create([], [output], funding_accounts, change_account)
+
```

Comment:
Can you reduce the number of blank lines and inline `hash_of_address` to be consistent with how all of the other `Transaction` methods are implemented.

Reply:
An address is already a hash of the public key. It's not useful to have this variable since nothing happens with it and it's just doing a common conversion.

Reply:
![image](https://user-images.githubusercontent.com/17583974/44466801-d23a0100-a63e-11e8-9b2c-acdc811738a5.png)

Reply:
We use `hexlify()` and `unhexlify()` all over the place and at least half of the usage is inline (doing the conversion just as the variable is passed to some function). This hex address to hash160 conversions is similar to `hexlify/unhexlify` and is pretty boring and standard operation. It doesn't need to be pulled out and highlighted to bring attention to it, if anything it distracts from the main flow of the function.

Reply:
`hexlify`/`unhexlify` is a built-in function which everyone knows about. But not the conversion for our hash. So this can be separate variable to make things clearer.